src/Nethermind/Nethermind.Evm.Test/CallTests.cs

```diff
@@ -0,0 +1,55 @@
+// SPDX-FileCopyrightText: 2025 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Specs;
+using NUnit.Framework;
+
+namespace Nethermind.Evm.Test
+{
+    public class CallTests : VirtualMachineTestsBase
+    {
+        protected override long BlockNumber => MainnetSpecProvider.ParisBlockNumber;
+        protected override ulong Timestamp => MainnetSpecProvider.OsakaBlockTimestamp;
+
+        [Test]
+        [TestCase(Instruction.CALL)]
+        [TestCase(Instruction.CALLCODE)]
+        [TestCase(Instruction.DELEGATECALL)]
+        [TestCase(Instruction.STATICCALL)]
+        public void Stack_underflow_on_call(Instruction instruction)
```

Comment:
This test fails on master. We need to repeat it in hive.
To be more clear: looks like consensus on execution is changed in this pr, and it's not revealed by tests common for ELs